pymc_marketing/mmm/transformers.py

```diff
@@ -826,10 +826,10 @@ def tanh_saturation_baselined(
     return gain * x0 * pt.tanh(x * pt.arctanh(r) / x0) / r
 
 
-def michaelis_menten(
+def michaelis_menten_function(
     x: float | np.ndarray | npt.NDArray[np.float64],
-    alpha: float | np.ndarray | npt.NDArray[np.float64],
-    lam: float | np.ndarray | npt.NDArray[np.float64],
+    alpha: float,
+    lam: float,
```

Comment:
I still don't know the best type hints here. But TensorVariable is also accepted here

Reply:
> I still don't know the best type hints here. But TensorVariable is also accepted here

Reverted changes to type hints